From b4a2facd62abcfdb4e2b8745929996de53bacc34 Mon Sep 17 00:00:00 2001
From: Paul Gortmaker <paul.gortmaker@windriver.com>
Date: Tue, 2 May 2017 18:21:12 -0400
Subject: [PATCH 055/221] soc/qbman: Move dma-mapping.h include to qman_priv.h
Git-commit: 290d638e04e7b94ff34d2f5e9426e7ce617e9a59
Patch-mainline: v4.12-rc1
References: FATE#320030

With the include there it fixes all build failures in this
directory caused by commit 461a6946b1f9.

Fixes: 461a6946b1f9 ("iommu: Remove pci.h include from trace/events/iommu.h")
Signed-off-by: Joerg Roedel <jroedel@suse.de>
Signed-off-by: Alexander Graf <agraf@suse.de>

---
 drivers/soc/fsl/qbman/qman_portal.c | 2 --
 drivers/soc/fsl/qbman/qman_priv.h   | 1 +
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/soc/fsl/qbman/qman_portal.c b/drivers/soc/fsl/qbman/qman_portal.c
index 9991cf8..c9a9bcb 100644
--- a/drivers/soc/fsl/qbman/qman_portal.c
+++ b/drivers/soc/fsl/qbman/qman_portal.c
@@ -30,8 +30,6 @@
 
 #include "qman_priv.h"
 
-#include <linux/dma-mapping.h>
-
 struct qman_portal *qman_dma_portal;
 EXPORT_SYMBOL(qman_dma_portal);
 
diff --git a/drivers/soc/fsl/qbman/qman_priv.h b/drivers/soc/fsl/qbman/qman_priv.h
index 22725bd..5fe9faf 100644
--- a/drivers/soc/fsl/qbman/qman_priv.h
+++ b/drivers/soc/fsl/qbman/qman_priv.h
@@ -33,6 +33,7 @@
 #include "dpaa_sys.h"
 
 #include <soc/fsl/qman.h>
+#include <linux/dma-mapping.h>
 #include <linux/iommu.h>
 
 #if defined(CONFIG_FSL_PAMU)
-- 
1.8.5.6

